Source/FullScreenPokemon.ts

```diff
@@ -1014,13 +1014,14 @@ module FullScreenPokemon {
          */
         keyDownSelect(thing: ICharacter, event?: Event): void {
             thing.FSP.ModAttacher.fireEvent("onKeyDownSelect");
+            thing.FSP.MathDecider.getConstant("items").SelectItem = thing.FSP.MathDecider.getConstant("items").Bicycle;
```

Comment:
No need for this. This PR will remove the default ability to start the bicycle with select.